tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx

```diff
@@ -93,6 +96,41 @@ using namespace ROOT::Detail::RDF;
 
 using Hist_t = ::TH1D;
 
+class RBranchSet {
+   std::vector<TBranch *> fBranches;
+   std::vector<std::string> fNames;
```

Comment:
```suggestion
   std::vector<std::pair<TBranch *, std::string> fBranches;
```
since the 2 must to be in sync. (and then using std::find_if and a lamba still allows the needed searches).

Reply:
fair enough